### ISSUE TITLE
Python API suggestion - nested dictionary as internal dictionary

### ISSUE THREAD

**Issue by danielhuppmann** [openclimatedata/globalwarmingpotentials#3]
I think that a nested dictionary would make more sense on the Python API, so that is easier to see which GWP-metrics exist... (Finding out which attributes exist seems a bit messy).

```python
_data = {
    SARGWP100 = {
        "CH4": 21.0,
    ...
}
```

and two public API functions:

```python
def metrics():
    return list(_data)

def get_values(metric):
    return _data[metric]
```

**Comment by znicholls** [comment]
+1 from me

**Comment by rgieseke** [comment]
A nested dict was my initial attempt as well, i liked the slightly simpler auto-completion (which is now great for dict keys as well, just a little bit more typing), but i agree that this is simpler.

See a proposal in #4 -- i don't think special API functions are necessary, one could use `.keys()` or `list(globalwarmingpotentials)` directly i think.

I've also added an optional Pandas Data Frame.

**Comment by rgieseke** [comment]
Closed in #4
